src/index.html

```diff
@@ -186,6 +186,29 @@ <h3 class="text-xl text-gray-50 font-semibold">
         </div>
       </div>
     </div>
+    <!--Smart Cells-->
+    <div
+      class="flex flex-col md:flex-row items-center space-y-8 md:space-y-0 md:space-x-8 lg:space-x-28"
+    >
+      <div class="flex flex-col align-ite mt-8">
+        <h3 class="text-xl text-gray-50 font-semibold">
+          &#9889; Smart cells
```

Comment:
```suggestion
          Smart cells
```

@josevalim please check the preview and let me know if you still want the icon. Alternatively we can use the remix icon, however I think we are better off without the icon, for consistency :)

Reply:
Anyway is fine. It is your call!

Reply:
In that case let's drop it :D